src/Service/LeantimeApiService.php

```diff
@@ -242,6 +388,155 @@ public function getPlanningData(): PlanningData
         return $planning;
     }
 
+    public function getTimesheetsForTicket($ticketId): mixed
+    {
+        return $this->request(self::API_PATH_JSONRPC, 'POST', 'leantime.rpc.timesheets.getAll', ['invEmpl' => '-1', 'invComp' => '-1', 'paid' => '-1', 'ticketFilter' => $ticketId]);
+    }
+
+    /**
+     * @throws ApiServiceException
+     */
+    private function getIssuesForProjectMilestone($projectId, $milestoneId): array
+    {
+        return $this->request(self::API_PATH_JSONRPC, 'POST', 'leantime.rpc.tickets.getAll', ['searchCriteria' => ['currentProject' => $projectId, 'milestone' => $milestoneId]]);
+    }
+
+    /**
+     * @throws ApiServiceException
+     * @throws \Exception
+     */
+    public function getSprintReportData(string $projectId, string $versionId): SprintReportData
+    {
+        $sprintReportData = new SprintReportData();
+        $epics = $sprintReportData->epics;
+        $issues = $sprintReportData->issues;
+        $sprints = $sprintReportData->sprints;
+
+        $spentSum = 0;
+        $remainingSum = 0;
+
+        $epics->set('noEpic', new SprintReportEpic('noEpic', 'Uden Tag'));
+
+        // Get version and project.
+        $milestone = $this->getMilestone($versionId);
+        $project = $this->getProject($projectId);
+
+        $issueEntries = $this->getIssuesForProjectMilestone($projectId, $versionId);
+
+        $issueCount = 1;
+        foreach ($issueEntries as $issueEntry) {
+            $issue = new SprintReportIssue();
+            $issues->add($issue);
+
+            // Set issue tag.
+            if (isset($issueEntry->tags)) {
```

Comment:
$issueEntry->tags seems to be an array but it is treated as a single string

Reply:
Tags are stored as a comma seperated string. In our implementation of Leantime, tickets are only supposed to have one tag, so i figured this would work out. Tickets with multiple tags would not break, but it would look wierd in the report.